tests/vm_test.go

```diff
@@ -284,11 +285,13 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		It("[test_id:3162]should ignore kubernetes and kubevirt annotations to VMI", func() {
-			vm := createVM(virtClient, libvmifact.NewCirros(
-				libvmi.WithAnnotation("kubevirt.io/test", "test"),
-				libvmi.WithAnnotation("kubernetes.io/test", "test"),
-			))
-			vm = startVM(virtClient, vm)
+			vm := libvmi.NewVirtualMachine(libvmifact.NewCirros(), libvmi.WithRunning())
+			vm.Annotations = map[string]string{
+				"kubevirt.io/test":   "test",
+				"kubernetes.io/test": "test",
+			}
+			vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, k8smetav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Test seems flaky, you will have to wait till VMI is created, IOW that the VM is ready
```
vm := libvmi.NewVirtualMachine(libvmifact.NewCirros(), libvmi.WithRunning())
vm.Annotations = map[string]string{
  "kubevirt.io/test":   "test",
  "kubernetes.io/test": "test",
}
			vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, k8smetav1.CreateOptions{})
Expect(err).ToNot(HaveOccurred())
Eventually(ThisVM(vm), 360*time.Second, 1*time.Second).Should(BeReady())
```

Reply:
Thanks Igor :)